lib/completion-manager.coffee

```diff
@@ -165,7 +165,7 @@ class CompletionManager extends LTool
     filedir = parsed_fname.dir
     filebase = parsed_fname.base  # name only includes the name (no dir, no ext)
 
-    labels = find_in_files(filedir, filebase, /\\label\{([^\}]+)\}/g)
+    labels = find_in_files(filedir, filebase, /\\label(?:\[[^\{\}]*\]){0,1}\{([^\}]+)\}/g)
```

Comment:
Way better, thanks. Indeed it is cleveref that must deal with cases like \label[{foo}]{bar}.